sass/directives/_headers.scss

```diff
@@ -60,39 +60,14 @@
   }
 }
 
-@mixin header-hero-product($background-image-large, $background-image-medium: nil) {
+@mixin header-hero {
   position: relative;
-  min-height: $header-height-product;
   overflow: hidden;
   background-color: $base-background-color;
-  background-image: url(asset_path(#{$background-image-large}));
-  background-size: auto $header-height-product;
-  background-position: right bottom;
-  background-repeat: repeat no-repeat;
-
-  @media only screen and (max-width: $medium-screen-max) {
-    background-image: url(asset_path(#{$background-image-medium}));
-  }
 
   @media only screen and (max-width: $small-screen-max) {
     @include display(flex);
     @include align-items(center);
-    min-height: unset;
```

Comment:
`min-height: unset` on mobile might be useful to all headers. I'm not saying you should or shouldn't remove it. I just want to know that you considered it and what are you thoughts on keeping it in the main mixin vs moving it to the product-specific mixin?